anime-list-master.xml

```diff
@@ -28346,6 +28346,9 @@
   </anime>
   <anime anidbid="10274" tvdbid="79214" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="">
     <name>Mushishi Tokubetsu Hen: Hihamu Kage</name>
+    <mapping-list>
+      <mapping anidbseason="1" tvdbseason="0">;1-2;</mapping>
+    </mapping-list>
```

Comment:
Couldn't the same be achieved by just setting "episodeoffset" to "1"?

Reply:
you're right, don't know if there is a prefered notation.